src/interface_adapter/finish_round/FinishRoundPresenter.java

```diff
@@ -40,8 +41,21 @@ public void prepareNextRoundView(FinishRoundOutputData outputData) {
         roundState.setGenre(outputData.getGenre());
         roundState.setCurrentLives(outputData.getLives());
         roundState.setUserAnswer("");
+
+        if (roundState.isEasyRound()) {
+            roundState.setMultipleChoice1(outputData.getMultipleChoiceAnswers().get(0));
+            roundState.setMultipleChoice2(outputData.getMultipleChoiceAnswers().get(1));
+        }
+        else if (roundState.isMediumRound()) {
+            roundState.setMultipleChoice1(outputData.getMultipleChoiceAnswers().get(0));
+            roundState.setMultipleChoice2(outputData.getMultipleChoiceAnswers().get(1));
+            roundState.setMultipleChoice3(outputData.getMultipleChoiceAnswers().get(2));
+            roundState.setMultipleChoice4(outputData.getMultipleChoiceAnswers().get(3));
+        }
```

Comment:
```suggestion
roundState.setMultipleChoiceOptions(createGameOutputData.getMultipleChoiceAnswers());
```
Again, we can simplify further with a single line.